pyinfra/facts/server.py

```diff
@@ -307,6 +307,43 @@ def process(self, output):
         return items
 
 
+class OsRelease(FactBase):
+    """
+    Returns a dictionary of release information stored in ``/etc/os-release``.
+
+    .. code:: python
+
+        {
+          "name": "EndeavourOS",
+          "pretty_name": "EndeavourOS",
+          "id": "endeavouros",
+          "id_like": "arch",
+          "build_id": "2024.06.25",
+          ...
+        }
+    """
+
+    def command(self):
+        return "cat /etc/os-release"
+
+    def process(self, output):
+        items = {}
+
+        for line in output:
+            if "=" not in line:
+                continue
+
+            key, value = line.split("=", 1)
+            key = key.strip().lower()
+
+            value = value.strip()
+            value = value.strip('"')
```

Comment:
Nitpick: You could chain these two calls to `strip()`

Reply:
I did even better :-)